011_mark_safe_L40/011_mark_safe_L40_unsafe.py

```
@@ -0,0 +1,43 @@
+from django.http import HttpResponse
+from django.template import loader
+
+def not_really_safe(request):
+    template = loader.get_template('contents.html')
+    not_actually_safe = mark_safe(
+        """
+        <div>
+            <p>Content! %s</p>
+        </div>
+        """ % request.POST.get("contents")
+    )
```

Ruff (0.14.1)

Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)